go.mod

```diff
@@ -3,6 +3,6 @@ module github.com/vishvananda/netlink
 go 1.12
 
 require (
-	github.com/vishvananda/netns v0.0.0-20200728191858-db3c7e526aae
```

Comment:
these changes should not be required for this PR.
Feel free to push them via a separate PR

Reply:
Removed the changes